### ISSUE TITLE
Bios settings?

### ISSUE THREAD

**Issue by Mannshoch** [TobleMiner/wintron7.0#6]
Do you have any suggestion for the Bios settings to change on this Tablet?

**Comment by DSchndr** [comment]
There is nothing useful to change.
By the way, never ever touch the usb settings in the bios.

Which firmware versions are on your tablet (first tab)?

**Comment by TobleMiner** [comment]
Yes, @DSchndr is right! Be super careful especially with the USB mux settings. Is is really easy to permanently disable the USB port preventing you from ever turning it on again.

I do agree, there is nothing useful you can change in the BIOS. The one and only thing you might need to touch is virtualization support. If you do for some reason plan on using virtual machines on the device you might have to enable it.

**Comment by Mannshoch** [comment]
Bios Version 2.16.1242
 - After shutdown I have to boot until black screen Press the power button long for a forced restart. then it boot into system
 - My Problem is also, that I do not get the touch screen run even I create the '/lib/firmware/silead/' directory and put the file into it. And in Bios there is an option to change a screen setting.
 - Also I hade huge Problems with Installing Linux. the only one which was nod crashed while installing was Q4OS (kernel 4.19) with trinity. Som Installer told me at least 1024MB Ram is needed other crashed while installing.
